hudi-aws/src/main/java/org/apache/hudi/config/HoodieAWSConfig.java

```diff
@@ -63,6 +63,11 @@ public class HoodieAWSConfig extends HoodieConfig {
         .sinceVersion("0.10.0")
         .withDocumentation("AWS session token");
 
+  public static final ConfigProperty<String> AWS_REGION = ConfigProperty
+          .key("hoodie.aws.region")
```

Comment:
Should this config be specific to Glue ? How are we planning to use this config for pointing to S3 in different region ? Will that be a separate config ?

Reply:
The original intent of adding this config was only for glue. If S3 and glue need to be pointed to different regions, then we need a separate config for S3.

Reply:
Moved this to GlueSyncConfig class.